packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -182,8 +178,8 @@ class _DropdownMenuState<T> extends State<_DropdownMenu<T>> {
         child: new Material(
           type: MaterialType.transparency,
           textStyle: route.style,
-          child: new ScrollConfiguration(
-            delegate: new _DropdownScrollConfigurationDelegate(Theme.of(context).platform),
+          child: new ScrollConfiguration2(
+            behavior: new _DropdownScrollBehavior(Theme.of(context).platform),
```

Comment:
is it more correct to get the Theme here, or to use the context given to the scroll behavior?